en/docs/release-notes.md

```diff
@@ -1,43 +1,48 @@
 # Release Notes
 
+## Dec 6, 2021
+### API updates
+1. Supported modifying acceleration regions of exiting transports.
+2. Supported managing transport shield by specifying regions.
+3. Added APIs to get service quotas, billing data, and resource summary.
```

Comment:
It would be nice to link to the documentation for each of the new APIs.

Reply:
Will add the link after APIs docs updated